cookbook/test/integration/environments/stage.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "stage",
+  "json_class": "Chef::Environment",
+  "chef_type": "environment"
+}
```

Comment:
Seems unnecessary… this can be omitted and Test Kitchen/Chef will assume you want to use the `_default` environment.